ghcup-vanilla-0.0.7.yaml

```diff
@@ -5804,6 +5803,101 @@ ghcupDownloads:
               dlUri: https://downloads.haskell.org/~hls/haskell-language-server-2.5.0.0/haskell-language-server-2.5.0.0-aarch64-apple-darwin.tar.xz
               dlSubdir: haskell-language-server-2.5.0.0
               dlHash: 2e5083ebf7fc9dd3c5aa31059f9336bec4407fffb21b93a20decb49e9cf880a4
+    2.6.0.0:
+      viTags:
+        - Latest
```

Comment:
Where is this documented?

Reply:
Nowhere, I guess.

Reply:
In all the previous patches I've made, I've always added exactly the same things to both the vanilla and default yamls, including updating or not updating the Recommended tag.

Reply:
Yes, we should enforce reviews. I hope @runeksvendsen will give us a hand with that.

Reply:
@hasufell I'm happy to assist in reviewing, but that requires that PRs be open for longer before being merged. Looks like this PR was merged less than 48 hours after being opened. I'm working part time on GHCup, so I don't have time every day to look at tickets/PRs.

Reply:
@runeksvendsen yes... I started to enforce review through branch protection rules.

Previously I had signalled that WT can raise and merge PRs on their own, since it wasn't clear whether I'll continue to maintain things on my own or at all. 

So no one is to blame here. We'll just switch things up from now on.

Reply:
> Previously I had signalled that WT can raise and merge PRs on their own, since it wasn't clear whether I'll continue to maintain things on my own or at all.

Who/what is WT?

Reply:
Well-Typed